src/sentry/api/bases/organizationmember.py

```diff
@@ -53,7 +53,7 @@ class OrganizationMemberEndpoint(OrganizationEndpoint):
     def convert_args(
         self,
         request: Request,
-        organization_slug: str,
+        organization_slug: int | str | None = None,
```

Comment:
why do you need `none`? looks like before this was not optional

Reply:
mypy was complaining Liskov Substitution Principle was being violated 😕 . I searched up what it meant, and it was because the inherited `OrganizationEndpoint`'s `convert_args` has org_slug's type as int | str | None = None. LSP says that the overriden method shouldn't have arguments more restrictive than the superclass. I am not sure why we didn't get this error before when `OrganizationMemberEndpoint` was modified.

i think by making this method less restrictive, we uphold the principle? we also don't have to worry about slug possibly being `None`, as the OrganizationEndpoint method checks for None and throws an exception.